MediaBrowser.Model/Session/TranscodeReason.cs

```diff
@@ -1,32 +1,46 @@
 #pragma warning disable CS1591
 
+using System;
+
 namespace MediaBrowser.Model.Session
 {
+    [Flags]
```

Comment:
Just as a note- changing this enum to `Flags` changed the integer representation of each value. This shouldn't matter as the API returns the string.

Reply:
yeah, i saw that the api was string based and I chose to change the values and have the sort-order be meaningful as well